src/Orchard.Web/Modules/Orchard.Users/Module.cs

```diff
@@ -29,11 +30,12 @@ public Startup(ShellSettings shellSettings, IOptions<IdentityOptions> options)
 
         public override void Configure(IApplicationBuilder builder, IRouteBuilder routes, IServiceProvider serviceProvider)
         {
+            builder.UseIdentity();
```

Comment:
I assume this line is fixing the Setup phase. We need to find out which service is missing from this call, we are aware that we should not have this line.